.travis.yml

```diff
@@ -7,6 +7,7 @@ go: 1.6
 install:
   - go get -u github.com/golang/lint/golint
   - go get -u github.com/Masterminds/glide
+  - go get -u golang.org/x/net/context
   - export GO15VENDOREXPERIMENT=1
```

Comment:
@ahmetalpbalkan Then we need to manually update the commit number in glide.lock and revert go-autorest lock file. Shall I do that?